spec/features/consumer_searches_remote_advice_spec.rb

```diff
@@ -37,13 +57,38 @@ def when_i_submit_a_search_selecting_online_advice
   end
 
   def then_i_am_shown_firms_that_provide_advice_online
-    expect(remote_results_page.firms).to be_present
+    expect(remote_results_page.firms.map(&:name)).
```

Comment:
We're all guilty of mapping over the firm name everywhere in the specs. It would be worth while pushing this into the results page as `firm_names` or similar, do you think?